.eslintrc.json

```diff
@@ -4,6 +4,7 @@
     "es6": true,
     "jest": true
   },
+  "parser": "babel-eslint",
```

Comment:
Is there a reason why this is added to the config? :thinking: 


Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/react-redux).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

Reply:
I have added parser: babel-eslint because eslint keeps giving me this error:
_**"Parsing error: unexpected token ="**_ for the first arrow function within my components' class and then does not check the rest of the code after that. It's the same issue reported [here](https://github.com/eslint/eslint/issues/10137) and the solution I saw is to add that.

Reply:
Could you please open a PR? Also add how to recreate the issue in the PR description. 🙏🏽

Reply:
Alright. I will do that.